### DISCUSSION TITLE
OPENSSL 1.1.1 instead of 3.0.0 ?

### DISCUSSION THREAD

**Discussion by Neb2653** [ggml-org/llama.cpp#8539] [Q&A]
Hello,

We are using RHEL 8.9 having OPENSSL 1.1.1. In order to build llamacpp server we need to have at least OPENSSL 3.0.0.

Is it possible to use OPENSSL 1.1.1 or some other method ?

Thanks

**Comment by ggerganov** [comment]
I think it depends on if `httplib` supports it: https://github.com/yhirose/cpp-httplib#ssl-support
